src/python/WMCore/MicroService/Unified/MSCore.py

```diff
@@ -36,16 +35,12 @@ def __init__(self, msConfig, logger=None):
                                    httpDict={'cacheduration': 1.0},
                                    logger=self.logger)
 
-        # hard code it to production DBS otherwise PhEDEx subscribe API fails to match TMDB data
-        dbsUrl = "https://cmsweb.cern.ch/dbs/prod/global/DBSReader"
-        if usingRucio():
-            # FIXME: we cannot use Rucio in write mode yet
-            # self.rucio = Rucio(self.msConfig['rucioAccount'], configDict={"logger": self.logger})
-            self.phedex = PhEDEx(httpDict={'cacheduration': 0.5},
-                                 dbsUrl=dbsUrl, logger=self.logger)
+        if self.msConfig['useRucio']:
+            self.rucio = Rucio(self.msConfig['rucioAccount'], configDict={"logger": self.logger})
         else:
-            self.phedex = PhEDEx(httpDict={'cacheduration': 0.5},
-                                 dbsUrl=dbsUrl, logger=self.logger)
+            # hard code it to production DBS otherwise PhEDEx subscribe API fails to match TMDB data
+            dbsUrl = "https://cmsweb.cern.ch/dbs/prod/global/DBSReader"
```

Comment:
I think I have tested it in the past with testbed DBSReader and it was giving me some failures (likely the reason for this short comment above).
It could go in the config as well, but given that it won't change between different instances of the service, I decided simply to hard-wire it here.